src/main/resources/application.properties

```diff
@@ -25,11 +25,10 @@ spring.flyway.schemas=document
 spring.jpa.properties.hibernate.temp.use_jdbc_metadata_defaults=false
 
 docs.queue.name=document-queue
-docs.queue=seda://${docs.queue.name}
+docs.queue=aws-sqs://arn:aws:sqs:eu-west-2:000000000000:${docs.queue.name}?amazonSQSClient=#sqsClient&messageAttributeNames=All
```

Comment:
Not sure it matters but do we need all the attribute names?

Reply:
I think Camel might use them but I'll check.